doc/lsst.fgcmcal/tasks/lsst.fgcmcal.fgcmFitCycle.FgcmFitCycleTask.rst

```diff
@@ -0,0 +1,125 @@
+
+Processing summary
+==================
+
+``FgcmFitCycleTask`` reads in the star observations and the look-up table, performs an atmosphere and instrument fit, and outputs fit parameters as well as a comprehensive list of QA plots.  If the config option ``isFinalCycle`` is set to ``True`` then additional datafiles are output that are used by :doc:`lsst.fgcmcal.fgcmOutputProducts.FgcmOutputProductsTask`.
+
+.. _lsst.fgcmcal.fgcmFitCycle.FgcmFitCycleTask-cli:
+
+fgcmFitCycle.py command-line interface
+======================================
+
+Note that no ``--id`` arguments are used by ``fgcmMakeLutTask``.
+
+.. code-block:: text
+
+   fgcmFitCycle.py REPOPATH [@file [@file2 ...]] [--output OUTPUTREPO | --rerun RERUN] [--configfile configfile] [other options]
+
+Key arguments:
+
+:option:`REPOPATH`
+   The input Butler repository's URI or file path.
+
+Key options:
+
+:option:`--configfile`:
+   The config file to use.  For all runs except the initial, this should be the config file output from the previous cycle.
+
+.. seealso::
+
+   See :ref:`command-line-task-argument-reference` for details and additional options.
+
+.. _lsst.fgcmcal.fgcmFitCycle.FgcmFitCycleTask-api:
+
+Python API summary
+==================
+
+.. lsst-task-api-summary:: lsst.fgcmcal.fgcmFitCycle.FgcmFitCycleTask
+
+.. _lsst.fgcmcal.fgcmFitCycle.FgcmFitCycleTask-butler:
+
+Butler datasets
+===============
+
+When run as the ``fgcmFitCycle.py`` command-line task, or directly through the `~lsst.fgcmcal.fgcmFitCycle.FgcmFitCycleTask.runDataRef` method, ``FgcmFitCycleTask`` obtains datasets from the input Butler data repository and persists outputs to the output Butler data repository.
+Note that configurations for ``FgcmFitCycleTask``, and its subtasks, affect what datasets are persisted and what their content is.
+
+.. _lsst.fgcmcal.fgcmFitCycle.FgcmFitCycleTask-butler-inputs:
+
+Input datasets
+--------------
+
+``camera``
+   Camera geometry and detector object
+``fgcmLookupTable``
+    FGCM look-up table produced by :doc:`lsst.fgcmcal.fgcmMakeLut.FgcmMakeLutTask`
+``fgcmVisitCatalog``
+    Catalog (`lsst.afw.table`) of visit metadata produced by :doc:`lsst.fgcmcal.fgcmBuildStars.FgcmBuildStarsTask`
+ ``fgcmStarObservations``
+    Catalog of star observations produced by :doc:`lsst.fgcmcal.fgcmBuildStars.FgcmBuildStarsTask`
+``fgcmStarIds``
+    Catalog of unique star ids, positions, and number of observations produced by :doc:`lsst.fgcmcal.fgcmBuildStars.FgcmBuildStarsTask`
+``fgcmStarIndices``
+    Catalog of indices linking unique star ids to star observations produced by :doc:`lsst.fgcmcal.fgcmBuildStars.FgcmBuildStarsTask`
+``fgcmReferenceStars``
+    Catalog of reference stars matched to unique star ids produced by :doc:`lsst.fgcmcal.fgcmBuildStars.FgcmBuildStarsTask`
+``fgcmFitParameters``
+    Catalog of fit parameters from previous fit cycle (if available)
+
+.. _lsst.fgcmcal.fgcmFitCycle.FgcmFitCycleTask-butler-outputs:
+
+Output datasets
```

Comment:
I feel the non-butler outputs (new configs and the QA plots) are important for this task but are missing here (reasonably). 

Would it be too messy to have a session of "non-butler outputs"?  Or perhaps expanding the intro to stress that?